modules/java-modules/elastic-v2/src/main/java/org/syslog_ng/elasticsearch_v2/client/ESTransportClient.java

```diff
@@ -63,6 +69,13 @@ protected Client createClient(List<Class<? extends Plugin>> plugins) {
 		transportClient = transportClientBuilder.build();
 		
 		addServersToClient(transportClient);
+
```

Comment:
Yeah, it strange because I factored out these lines to a method(with an IDE...) .
I'll fix this.